pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "MM-classification-service"
+version = "0.1.0"
+description = ""
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.8"
+affine = "2.3.1"
+apispec = "6.0.0"
+apispec-webframeworks = "0.5.2"
+attrs = "22.1.0"
+cachetools = "5.2.0"
+certifi = "2023.7.22"
+charset-normalizer = "3.1.0"
+click = "8.1.3"
+click-plugins = "1.1.1"
+click-spinner = "0.1.10"
+cligj = "0.7.2"
+dash = "2.7.0"
+dash-core-components = "2.0.0"
+dash-html-components = "2.0.0"
+dash-table = "5.0.0"
+distlib = "0.3.6"
+docopt = "0.6.2"
+filelock = "3.8.0"
+Flask = "2.2.5"
+Flask-Cors = "3.0.10"
+Flask-Sock = "*"
+idna = "3.4"
+importlib-metadata = "5.0.0"
+itsdangerous = "2.1.2"
+Jinja2 = "3.1.2"
+joblib = "1.2.0"
+MarkupSafe = "2.1.1"
+marshmallow = "3.18.0"
+mercantile = "1.2.1"
+numpy = "1.23.4"
+packaging = "21.3"
+pandas = "1.5.1"
+Pillow = "10.0.1"
+pipenv = "2022.10.25"
+pipreqs = "0.4.13"
+platformdirs = "2.5.2"
+plotly = "5.11.0"
+pyparsing = "3.0.9"
+python-dateutil = "2.8.2"
+pytz = "2022.6"
+PyYAML = "6.0"
+rasterio = "1.3.3"
+requests = "2.31.0"
+Shapely = "1.8.5.post1"
+six = "1.16.0"
+snuggs = "1.4.7"
+tenacity = "8.1.0"
+terracotta = "0.7.5"
+toml = "0.10.2"
+tqdm = "4.64.1"
+urllib3 = "1.26.14"
+virtualenv = "20.16.6"
+virtualenv-clone = "0.5.7"
+Werkzeug = "2.2.3"
+yarg = "0.1.9"
+zipp = "3.10.0"
```

Comment:
Does this accurately reflect the true top-level requirements of this package?  This smells like a `pip freeze` dump!

Reply:
Most of these dependencies aren't being used right now, but Jason mentioned that they will be required later on when he onboards code from the ML side onto the classification service. 

Is it okay if we leave this as is temporarily? Currently server.py only really uses flask.

Reply:
Leaving these in may cause very difficult to resolve conflicts later on!  @Jasonnyang @zclawr please make note of this before moving forward.  This may also cause a lot of difficulty in building this outside of `docker`!